app/services/push_notifications/broadcast.rb

```diff
@@ -0,0 +1,28 @@
+module PushNotifications
+  class Broadcast
+    def initialize(push_notifications:)
+      @push_notifications = push_notifications
+    end
+
+    def send
+      return unless push_notifications.any?
```

Comment:
You told me `Exponent::Push::Client` does not check this and so it blows up if `notifications` is empty, right? It would make things simpler if they did.

Reply:
Yep, but they don't :(